### ISSUE TITLE
Error Starting on "Worker"

### ISSUE THREAD

**Issue by krishnakolluneocol** [SFDO-Tooling/MetaDeploy#3498]
Getting this error when starting worker on a basic dyno

mkdir: cannot create directory ‘/opt’: Read-only file system

**Comment by davidmreed** [comment]
Hi @krishnakolluneocol. Hard to diagnose this issue without more information. Can you share a detailed summary of the setup you have done so far, and what action leads to this error?

**Comment by krishnakolluneocol** [comment]
I just deployed the MetaDeploy app to Heroku using the deploy button, changed the configuration for all dynos to basic dynos, and tried to start all dynos. I saw this error in the logs. Practically speaking, it doesn't seem to be affecting anything. 

I disabled the worker dyno and now just have "web" and "devworker" dynos running. Is there any documentation around what these different dynos are responsible for? I am able to get installs working with just a "web" and "devworker" dyno running, but I'm not sure if I'm missing something by having the other dynos "worker" and "worker-short" disabled
